quickjs.h

```diff
@@ -86,47 +86,12 @@ typedef struct JSRefCountHeader {
 } JSRefCountHeader;
 
 #define JS_FLOAT64_NAN NAN
+#define JSValueConst JSValue /* For backwards compatibility. */
```

Comment:
My plan is to follow up with a s/JSValueConst/JSValue/ PR and leave this to avoid breaking downstream users' builds.